src/main.rs

```diff
@@ -44,8 +54,8 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         let _ = manager.run().await;
     });
 
-    // Do other things or go to wait forever
-    pending::<()>().await;
-
-    Ok(())
+    signal::ctrl_c().await?;
```

Comment:
Modified this because sometimes when exiting gamescope-dbus with ctl_c, the dbus interface name was stuck registered, seemed like the process wasn't exiting properly